corehq/apps/export/models/new.py

```diff
@@ -1285,6 +1317,29 @@ def get_headers(self, **kwargs):
         return headers
 
 
+class MultiMediaExportColumn(ExportColumn):
+    """
+    A column that will take a multimedia file and transform it to the absolute download URL.
+    If transform_dates is set to True it will render the link with Excel formatting
+    in order to make the link clickable.
+    """
+
+    def get_value(self, domain, doc_id, doc, base_path, transform_dates=False, **kwargs):
+        value = super(MultiMediaExportColumn, self).get_value(domain, doc_id, doc, base_path, **kwargs)
+
+        if not value:
+            return value
+
+        download_url = u'{url}?attachment={attachment}'.format(
+            url=absolute_reverse('download_attachment', args=(domain, doc_id)),
+            attachment=value,
+        )
+        if transform_dates:
+            download_url = u'=HYPERLINK("{}")'.format(download_url)
```

Comment:
in excel, `HYPERLINK` takes a second arg which is the human readable name of the link. curious if you think it would be better to have just the filename be the human readable name